lib/ble.toit

```diff
@@ -210,6 +220,19 @@ class RemoteCharacteristic extends RemoteReadWriteElement_ implements Attribute:
   constructor.private_ .service .uuid .properties characteristic:
     super service characteristic
 
+  /**
+  Closes this characteristic.
+
+  Automatically removes the characteristic from the list of discovered characteristics of
+    the service.
+  */
+  close:
+    descriptors := discovered-descriptors
```

Comment:
It feels a bit icky that the field is public. Maybe the field should be private and we should have a getter that returns a copy of the private list?

Reply:
done.